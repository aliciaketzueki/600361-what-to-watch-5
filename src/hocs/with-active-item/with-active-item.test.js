import React, {Fragment} from "react";
import renderer from "react-test-renderer";
import withActiveItem from "./with-active-item";
import PropTypes from "prop-types";

const MockComponent = (props) => {
  const {children} = props;

  return (
    <div>
      {children}
    </div>
  );
};

MockComponent.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired,
};

const MockComponentWrapped = withActiveItem(MockComponent);

describe(`Render withActiveItem`, () => {
  it(`Should withActivePlayer render correctly`, () => {
    const tree = renderer.create(
        <MockComponentWrapped>
          <Fragment />
        </MockComponentWrapped>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
