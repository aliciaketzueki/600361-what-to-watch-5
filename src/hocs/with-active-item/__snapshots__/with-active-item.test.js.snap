// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render withActiveItem Should withActivePlayer render correctly 1`] = `<div />`;
